zkevm-circuits/src/evm_circuit/util.rs

```diff
@@ -499,10 +499,179 @@ impl<F: FieldExt, const N: usize> Expr<F> for RandomLinearCombination<F, N> {
     }
 }
 
-pub(crate) type Word<F> = RandomLinearCombination<F, 32>;
+#[derive(Clone, Debug)]
+pub(crate) struct WordLimbs<T, const N: usize> {
```

Comment:
I found all the `Word<F>` became `Word<Cell<F>>`. Wondering if we could add `Cell` in `WordLimbs` just like original `RandomLinearCombination` does. The code would look more succint, then it'll look like,
```
pub(crate) struct WordLimbs<T, const N: usize> {
    pub limbs: [T; N],
    pub(crate) cells: [Cell<F>; N],
}

Reply:
In new `Word` generic type, there is no single expression to represent multiple cells via RLC. So `RandomLinearCombination ` concept is no longer be used here.

Besides, I try to avoid putting limbs and cells together, I think it somehow violate pattern for single-responsibility of a class definition.

But, naming shorter is indeed better!! so i add few type alias to try make common type more shorter comfortable :P